webpack.config.js

```diff
@@ -2,10 +2,18 @@ const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const webpack = require('webpack');
 const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
+const dotenv = require('dotenv');
+
+dotenv.config();
 
 const isProd = process.env.NODE_ENV === 'production';
 const isTest = process.env.NODE_ENV === 'test';
 
+const envKeys = Object.keys(process.env).reduce((prev, cur) => {
```

Comment:
Please do not do this. It means ALL system envs will be encoded into JSON on frontend.
DefinePlugin must explicitly list every single env var we want to expose. No automation.

Reply:
removed